src/util/address.rs

```diff
@@ -64,7 +65,9 @@ pub enum Error {
     /// Script version must be 0 to 16 inclusive
     InvalidWitnessVersion(u8),
     /// Unable to parse witness version from string or opcode value
-    UnparsableWitnessVersion,
+    UnparsableWitnessVersion(ParseIntError),
+    /// Bitcoin script opcode does not match any known witness version, the script if malformed
+    MalformedWitnessVersion,
```

Comment:
Would it make sense to report the opcode or (probably more importantly) position within script?

Reply:
Not trivial, since this error is also produced by instructions, not only opcode. And the position is always first byte, according to SegWit